plugins/available/basher.plugin.bash

```diff
@@ -0,0 +1,14 @@
+cite about-plugin
+about-plugin 'initializes basher, the shell package manager'
```

Comment:
Can you use double quotes here as well?

Reply:
No problem, but is there a reason? I've taken to defaulting to single quotes when not doing expansion because it's faster for bash. Granted, the gain is small, but it's there.

Reply:
I see this is the standard in the project, so I resolve. I think we need to maybe clarify that in the DEVELOPMENT document